OC4IDS_Database_Data_Import.ipynb

```diff
@@ -105,25 +105,25 @@
         "  'mexico_cost_jalisco': 'http://www.costjalisco.org.mx/jsonprojects',\n",
         "  'ghana_cost_sekondi_takoradi': 'https://costsekondi-takoradigh.org/uploads/projectJson.json',\n",
         "  'mexico_nuevo_leon': 'http://si.nl.gob.mx/siasi_ws/api/edcapi/DescargarProjectPackage',\n",
-        "  'indonesia_cost_west_lombok': 'https://intras.fllajlombokbaratkab.or.id/oc4ids',\n",
+        "  'indonesia_cost_west_lombok': 'https://intras.lombokbaratkab.go.id/oc4ids',\n",
         "  'ukraine_cost_ukraine': 'https://portal.costukraine.org/data.json',\n",
-        "  'uganda_gpp': 'https://gppflattener.app/download?format=json',\n",
+        "  'uganda_gpp': 'https://gpp.ppda.go.ug/adminapi/public/api/open-data/v1/infrastructure/projects/download?format=json',\n",
         "  'malawi_cost_malawi': 'https://ippi.mw/api/projects/query'\n",
         "}\n",
         "\n",
         "if source_id == 'malawi_cost_malawi':\n",
-        "  # The IPPI API accepts a POST request JSON-encoded start_date and end_date paramters\n",
+        "  # The IPPI API accepts a POST request with JSON-encoded start_date and end_date paramters\n",
         "  payload = {\n",
         "      \"start_date\": \"2010-01-01\",\n",
         "      \"end_date\": datetime.today().strftime('%Y-%m-%d')\n",
         "    }\n",
         "  response = requests.post(sources[source_id], json=payload)\n",
-        "  \n",
+        "\n",
         "  with open('project_package.json', 'wb') as f:\n",
         "    f.write(response.content)\n",
         "\n",
         "else:\n",
-        "  response = requests.get(sources[source_id])\n",
+        "  response = requests.get(sources[source_id], verify=False)\n",
```

Comment:
@duncandewhurst Is it fine to leave verify as FALSE? I assume it helps when there are expired certificates or other issues?

Reply:
Yep, it's needed because there is an issue with the certificate for West Lombok.